rpip-template.md

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
A little confused on this file and the rename of the one in the RPIPs directory. Can you clarify intent?

Reply:
They're both the same, I was going to say we can decide which one to keep (which location is more intuitive) and delete the other

Reply:
I'll remove the one in the rpip folder

Reply:
That works for me